go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/go-logr/logr v0.1.0
 	github.com/go-openapi/spec v0.19.7
 	github.com/imdario/mergo v0.3.8
-	github.com/kubevirt/cluster-network-addons-operator v0.3.1-0.20200715072547-9876b6dfad40
+	github.com/kubevirt/cluster-network-addons-operator v0.3.1-0.20200811082954-d0b735298e44
```

Comment:
no urgent for now, but is there a chance that you switch to semver reflecting the actual version, like in other components?

Reply:
I promised that to @tiraboschi couple of months ago ... I'm down to switch it, I'm just a little anxious to do that since it may break the CSV generation which is hard to debug. We went for v-less version since CSV was not accepting the trailing V very well. I'm just afraid of my own scripts. Will try

Reply:
I opened a PR for it on CNAO master: https://github.com/kubevirt/cluster-network-addons-operator/pull/501

It won't be backported though, so it won't help in this PR.